regtests/0231_cert_status/create-certificates

```diff
@@ -58,7 +58,7 @@ openssl ca -in aws-server.csr -days $DAYS -cert CA-srv.crt \
 
```

Comment:
The comment at the top of this file seems truncated. Also "for mo than" I suppose should be "for more than" ?

Reply:
depend on certificate. Certificates for this test extended by the script

Reply:
The number of days the certificates going to be valid defined on siging this way 
openssl ca -in aws-client.csr -days $DAYS -cert CA-clt.crt \
  -keyfile CA-clt.key -batch -out aws-client.crt

Reply:
Sure, but you did not regenerate the certificate I suppose, you just extended it, no?

Reply:
I use to regenerate it completely. Maybe we can keep the same certificate request file. But I did not try it. I think no need to keep it.

Reply:
As I understanr the procedure. User creates certificate request file and related pair of keys. Sending certificate request with public key to authority. Authority generates signed certificates and sending it back to certificate user. User installing the certificate to own server.

Reply:
The comment in file was completely wrong, looks like it was part of script where it is copied from. I'll think what to do with it.

Reply:
Comment fixed.